### ISSUE TITLE
Feature Request: Automatically switch to current day when day changes

### ISSUE THREAD

**Issue by BurnyBoi** [lfos/calcurse#135]
I notice that if I leave calcurse running for more than a day, the calendar will always keep the currently selected date, and won't change. I then have to manually move the calendar to the current date to see my appointments every day.  I would like to request some way to automatically have calcurse select the current day as it changes. This way the current date's appointments will always be visible when the user returns to look at calcurse, even if they leave it running overnight.

**Comment by lfos** [comment]
I don't think this is a good idea as it might yield unexpected behavior when interacting with the user interface at midnight (i.e. if the day changes just a couple of milliseconds before adding a new item, the item will be added to the wrong day).

**Comment by lhca** [comment]
Yes, it should definitely not happen automatically at midnight. Changing the selected day behind the user's back will have unexpected consequences.

But there is another way. And one that is very easily implemented. Let calcurse catch a signal, set a flag and then set the selected day in the main input loop. A similar behaviour exists already for SIGUSR1 and data reload.

**Comment by lfos** [comment]
The problem with signals is that there is only a very limited amount of custom signals we can use and they are fairly limited in functionality as well (no easy way to pass additional parameters). If we want to add futher remote control features, we should probably think about using a more suitable technology (e.g. sockets) as well.

**Comment by Spacewalker2** [comment]
Maybe this can simply switched on and off and everyone can use it in the way he or she prefers it. Another solution would be a warning if anybody tries to create an appointment around midnight. I use calcurse in a tmux split screen and it is a little bit annoying that I have to jump every morning to the this screen and press ^g and the jump back. Here tmux runs 24/7 on a raspberry pi. But nevertheless thanks for the great work!

**Comment by lfos** [comment]
Another idea: we could detect whether the user is inactive (i.e. no prompt is open and there was no key input for a certain number of minutes). Then have an configurable feature to switch the current day on midnight only if the user is inactive.

**Comment by Spacewalker2** [comment]
This works for me.
